Packs/Cybereason/Integrations/Cybereason/Cybereason.py

```diff
@@ -1381,6 +1630,316 @@ def logout():
+            suspect_file, timeout_seconds)
+        new_malop_comments.append({"isSuccess": False, "message": malop_comment})
+    return new_malop_comments
+
+
+def fetchfile_progress():
+    return http_request('GET', '/rest/fetchfile/downloads/progress')
+
+
+def download_fetchfile_command():
+    batch_id = demisto.getArg('batchID')
+    demisto.log('Downloading the file with this Batch ID: {}'.format(batch_id))
+    response = download_fetchfile(batch_id)
+    if response.status_code == 200:
+        file_download = fileResult('download.zip', response.content)
+        demisto.results(file_download)
+    elif response.status_code == 500:
+        demisto.results('The given Batch ID has expired')
+    else:
+        demisto.results('Your request failed with the following error: ' + response.content + '. Response Status code: ' + str(
+            response.status_code))
+
+
+def download_fetchfile(batch_id: str) -> Any:
+    url = '/rest/fetchfile/getfiles/{batch_id}'.format(batch_id=batch_id)
+    return http_request('GET', url, custom_response=True, return_json=False)
+
+
+def close_fetchfile_command():
+    batch_id = demisto.getArg('batchID')
+    response = close_fetchfile(batch_id)
+    try:
+        if response.json()['status'] == 'SUCCESS':
+            demisto.results('Successfully aborts a file download operation that is in progress.')
+    except Exception:
+        raise Exception('The given Batch ID does not exist')
+
+
+def close_fetchfile(batch_id: str) -> Any:
+    url = '/rest/fetchfile/close/{batch_id}'.format(batch_id=batch_id)
+    return http_request('GET', url, custom_response=True, return_json=False)
+
+
+def malware_query_command():
+    needs_attention = demisto.getArg('needsAttention')
+    malware_type = demisto.getArg('type')
+    malware_status = demisto.getArg('status')
+    time_stamp = demisto.getArg('timestamp')
+    limit_range = demisto.getArg('limit')
+    limit_range = int(limit_range)
+    if limit_range > 0:
+        filter_response = malware_query_filter(needs_attention, malware_type, malware_status, time_stamp, limit_range)
+        demisto.results(filter_response)
+    else:
+        raise Exception("Limit cannot be zero or a negative number.")
+
+
+def malware_query_filter(needs_attention: str, malware_type: str, malware_status: str, time_stamp: str, limit_range: int) -> dict:
+    query = []
+    if bool(needs_attention) is True:
+        query.append({"fieldName": "needsAttention", "operator": "Is", "values": [bool(needs_attention)]})
+    if bool(malware_type) is True:
+        types = malware_type.split(",")
+        query.append({"fieldName": "type", "operator": "Equals", "values": types})
+    if bool(malware_status) is True:
+        is_status = malware_status.split(",")
+        query.append({"fieldName": "status", "operator": "Equals", "values": is_status})
+    if bool(time_stamp) is True:
```

Comment:
```suggestion
    if time_stamp:
```

By replacing it, every non-empty value will count as False.

Reply:
Resolved.